xbmc/cores/VideoPlayer/DVDCodecs/Video/MMALCodec.h

```diff
@@ -80,7 +80,7 @@ class CMMALVideo : public CDVDVideoCodec
   virtual void Reset(void);
   virtual bool GetPicture(DVDVideoPicture *pDvdVideoPicture);
   virtual bool ClearPicture(DVDVideoPicture* pDvdVideoPicture);
-  virtual unsigned GetAllowedReferences() { return 3; }
+  virtual unsigned GetAllowedReferences() { return NUM_BUFFERS; }
```

Comment:
I don't think so. VAAPI, VDPAU use 5. I would set the desired number explicitly in info.optimal_buffer_size instead of relying on a define that is subject to change.